src/main/java/com/kirinpatel/sync/gui/VLCJMediaPlayer.java

```diff
@@ -54,10 +54,10 @@
         media  = new Media("");

```

Comment:
I don't think the MediaPlayer has to be passed around, but after looking the refactor probably constitutes its own issue. Added #48